data.sql

```diff
@@ -1,6 +1,43 @@
 
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (1, 'Agumon', '02/03/20', 0, true,10.23);
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', '15/11/18',2, true,8);
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (3, 'Pikachu', '01/07/21', 1, false,15.04);
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (4, 'Davimon', '05/12/17', 5, true,11);
\ No newline at end of file
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (1, 'Agumon', '02/03/20', 0,true,10.23);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', '11/15/18', 2,true,8);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (3, 'Pikachu', '01/7/21', 1,false,15.04);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (4, 'Davimon', '05/12/17', 5,true,11);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (5, 'Chamandar', '02/08/20', 0, false, -11);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (6, 'Plantmon', '11/15/21', 2, true, -5.7);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (7, 'Squirtle', '04/02/93', 3, false, -12.13);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (8, 'Angemon', '06/12/05', 1, true, -45);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (9, 'Boarmon', '06/07/05', 7, true, 20.4);
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (10, 'Blossom', '10/13/98', 3, true, 17);\
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (11, 'Ditto', '05/14/22', 4, true, 22);
+
+-- Insert data into the owners table
+INSERT INTO owners (full_name, age)
+VALUES 
+    ('Sam Smith', 34),
+    ('Jennifer Orwell', 19),
+    ('Bob', 45),
+    ('Melody Pond', 77),
+    ('Dean Winchester', 14),
+    ('Jodie Whittaker', 38);
+
+-- Insert data into the species table
+INSERT INTO species (name)
+VALUES 
+    ('Pokemon'),
+    ('Digimon');
+
+-- Update the animals table to include species and owner information
+UPDATE animals SET 
+    species_id = CASE 
+                    WHEN name LIKE '%mon' THEN 2 
+                    ELSE 1 
+                 END,
+    owner_id = CASE 
+                    WHEN name = 'Agumon' THEN 1 
+                    WHEN name IN ('Gabumon', 'Pikachu') THEN 2 
+                    WHEN name IN ('Devimon', 'Plantmon') THEN 3 
+                    WHEN name IN ('Charmander', 'Squirtle', 'Blossom') THEN 4 
+                    WHEN name IN ('Angemon', 'Boarmon') THEN 5 
+                 END;
```

Comment:
- [ ] Kindly add a command that updates the species_id column of your animals' table. If an animal's name ends with mon it should have Digimon as a species_id, all other animals should have Pokemon instead. Add the command to your data.sql file.

Reply:
![Screenshot 2023-05-04 230236](https://user-images.githubusercontent.com/109290913/236349284-e709d721-dfe8-4997-a1d6-034205a4c575.png)


![Screenshot 2023-05-04 230323](https://user-images.githubusercontent.com/109290913/236349295-2e682cad-a9db-4040-b762-f7d7058b4bdb.png)

Reply:
Thank you very much for pointing out all I omitted @Kamba56

Reply:
Screenshots should be updated within the PR description.